### ISSUE TITLE
Add place autocomplete

### ISSUE THREAD

**Issue by chashmeetsingh** [openSUSE/osem#1356]
Using google place auto complete we can get the location details such as longitude, latitude, complete address automatically and use them instead for the `Venue`. It's a bad experience asking users to input latitude and longitude at least.

**Comment by siaw23** [comment]
how about getting rid of the long and lat portion and just autocompleting the country using geocode (or something similar) to autocomplete the country field? The address is enough info for the venue of a conference.

**Comment by chashmeetsingh** [comment]
The latitude and longitude can be used to show markers on maps

**Comment by jaels** [comment]
Hey, I'm pretty new to contributing to open source projects, and would like to gain more experience in that. Can I help with this issue? Thanks

**Comment by differentreality** [comment]
Hey @jaels, yes you can go ahead, no else seems to be working on it.

**Comment by jaels** [comment]
Hey @differentreality , I downloaded everything and opened the website on my machine, but there isn't much there. It looks like this:
![image](https://cloud.githubusercontent.com/assets/19172728/24295705/5b13af4c-109c-11e7-8bdc-594e90593268.png)
Where is the place autocomplete etc? should I sign up?

**Comment by differentreality** [comment]
Yes you should! Then you can create a new conference or import some from our test data
`rake data:test`

**Comment by jaels** [comment]
So, should I add google auto complete for the address, and cancel the latitude and longitude?

**Comment by differentreality** [comment]
@jaels you should check if google auto complete is going to work. We also need to show the venue on the map in the splashpage of the conference. Let us know what you try out with a Pull Request.

**Comment by jaels** [comment]
@differentreality Now I see that this part is written in Ruby. My language is Javascript... Do you know of any Junior bug in Javascript?

**Comment by differentreality** [comment]
Well yes, this is a ruby on rails application :-)

**Comment by jaels** [comment]
Ohh. My mistake. I do see there are some JS files...

**Comment by neilhalligan** [comment]
Is it ok to work on this?

**Comment by Abhinand-p** [comment]
@differentreality 
Hi,
I am working on this issue. I have included **geocoding code** in order to eliminate latitude and longitude entries. I am trying to resolve the errors. 

![image](https://cloud.githubusercontent.com/assets/12455261/24658169/8d86f72c-1966-11e7-9908-67c63b553c29.png)

![image](https://cloud.githubusercontent.com/assets/12455261/24656881/dd664fae-1961-11e7-875f-e91dcda1a1df.png)

![image](https://cloud.githubusercontent.com/assets/12455261/24656898/ea11bf22-1961-11e7-978c-04fc00159027.png)

Am I in the right path.

**Comment by Abhinand-p** [comment]
@differentreality 
Hi,
I am successful in eliminating latitude and longitude entries from the create venue form. I have executed in localhost. The geocoder package is installed. Based on the address specified, latitude and longitude values are automatically generated and entered into the Venues database. The map is now displayed based on the computed values. The following snapshots depict successful execution:

![image](https://cloud.githubusercontent.com/assets/12455261/24705220/8ec707a4-1a28-11e7-9176-6c144b474dae.png)
![image](https://cloud.githubusercontent.com/assets/12455261/24705226/956faffc-1a28-11e7-9db2-ab5034216a69.png)
Active record entries:
![image](https://cloud.githubusercontent.com/assets/12455261/24705239/a19793d0-1a28-11e7-86b5-4a12c9514f33.png)

Please comment.
